spec/script/create_written_spec.rb

```diff
@@ -0,0 +1,121 @@
+require Rails.root.join('script', 'create_written.rb').to_s
```

Comment:
Ooh, this could be a nice way to test more complicated logic our migrations! Nice!

Reply:
Yeah, Marri found an article about it! https://alexharv074.github.io/2016/04/23/rspec-testing-a-simple-ruby-script.html